docs/app/page.tsx

```diff
@@ -32,191 +34,208 @@ const SectionLink = ({
 
 // Page
 // ---------------
-export const Page = () => (
-  <main className="grid justify-center px-[--page-padding-md] xl:px-[--page-padding-xl]">
-    {/* Hero */}
-    <div className="grid h-[calc(100dvh-var(--page-header-height))] max-w-screen-lg place-items-center text-center">
-      <div className="-mx-[--page-padding-md] md:mx-0">
-        <h1 className="text-balance text-6xl font-extrabold lg:text-8xl">
-          Cultivate beautiful user interfaces
-        </h1>
-        <p className="text-text-primary/60 text-balance pb-10 pt-4 text-lg font-light leading-relaxed lg:pb-24 lg:pt-10 lg:text-xl">
-          Marigold is a design system for Reservix, providing components and
-          tools that let product teams focus on core challenges while creating
-          unified, accessible applications.
-        </p>
-        <Center>
-          <Inline data-toggle data-toggle-target="#foobar" space={6}>
-            <Link variant="primary" size="large" href="/getting-started">
-              <span className="inline-block translate-y-px">Get started</span>
+export const Page = () => {
+  const latestPost = getLatestPost();
+  return (
+    <main className="grid justify-center px-[--page-padding-md] xl:px-[--page-padding-xl]">
+      {/* Hero */}
+      <div className="grid h-[calc(100dvh-var(--page-header-height))] max-w-screen-lg place-items-center text-center">
+        <div className="-mx-[--page-padding-md] md:mx-0">
+          <Center>
+            <Link href={`/${latestPost.slug}`} variant="shiny" size="xsmall">
+              <AnimatedShinyText className="transition ease-out">
+                🚀✨ Read last updates!
```

Comment:
```suggestion
                🚀✨ Read lastest updates!
```

Reply:
I think this is the "more correct" english?

Reply:
latest